.env

```diff
@@ -5,7 +5,17 @@ SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/
 SIMORGH_ATI_BASE_URL=https://logws1363.ati-host.net?
 SIMORGH_CONFIG_URL=https://config.test.api.bbci.co.uk/
 SIMORGH_INCLUDES_BASE_URL=https://www.test.bbc.com/ws/includes
+SIMORGH_INCLUDES_BASE_AMP_URL=https://news.test.files.bbci.co.uk
```

Comment:
Should reset this file if possible - I think it causes some headaches for other devs if a change here is merged to latest as it's not supposed to be in the index

If it keeps showing up in your changes, you can try this command after resetting it on your branch:

`git update-index --assume-unchanged .env`

Reply:
I haven't seen that pesky .env for a while. I'll reset it.